cli/tools/mod.ts

```diff
@@ -0,0 +1,225 @@
+/// <reference lib="deno.ns" />
+import { join, dirname } from "https://deno.land/std@0.220.1/path/mod.ts";
+import { red, yellow, green } from "https://deno.land/std@0.220.1/fmt/colors.ts";
+import type { ToolResponse } from "../types.d.ts";
+
+interface CommandConfig {
+  desc: string;
+  args: readonly string[];
+}
+
+// Define allowed commands and their descriptions
```

Comment:
This is a nice data structure for these commands. At some point we can combine with the Cline.ALLOWED_AUTO_EXECUTE_COMMANDS allowlist.